Animations/Animation.js

```diff
@@ -37,20 +22,36 @@ var makeTransition = function (displayName, transitions) {
       // componentDidEnter FUNCTION: use to define actions to run after an animation has entry completed
       // componentDidLeave FUNCTION: use to define actions to run after an animation has leave completed
   return React.createClass({
+    // setAnimationDomNode FUNCTION: function to handle manipulating and setting a TransitionGroup in the DOM
+      //@param action STRING: the transition key you are looking to affect
+      //@param callback FUNCTION: callback attribute passed in from the containing function to act on function completion
+    setAnimationDomNode: function (action, callback) {
+      this.el = this.getDOMNode();
+      this.$el = $(this.el);
```

Comment:
Where are we defining `$`.  Does our animations library load jQuery?  Is there anyway we could use base Javascript to avoid this?